### DISCUSSION TITLE
What tool is used for documentation?

### DISCUSSION THREAD

**Discussion by ddahan** [python-poetry/poetry#5489] [General] [answered]
I just love how beautiful the Poetry documentation looks.
May I know what is the tool / theme used to build it?
Thanks.

**Comment by abn** [comment, marked as answer]
https://twitter.com/SDisPater/status/1440801419376795648

**Reply by ddahan** [reply]
Thanks! The result is so sleek that I didn't consider it could have been built "manually"! Good job.
